spring-cloud-config-server/src/main/java/org/springframework/cloud/config/server/environment/JdbcEnvironmentRepository.java

```diff
@@ -61,7 +59,15 @@
 	public JdbcEnvironmentRepository(JdbcTemplate jdbc) {
 		this.jdbc = jdbc;
 	}
-	
+
+	public JdbcEnvironmentRepository(JdbcTemplate jdbc, JdbcEnvironmentProperties properties) {
+		this.jdbc = jdbc;
+		if (properties != null) {
+			this.order = properties.getOrder() == null ? this.order : properties.getOrder();
```

Comment:
Just set it, no need for null check.

Reply:
Would you prefer to see the defaults for all fields on the environment repositories set in the properties classes?

Reply:
All, I forgot there are defaults, but those should now go in the properties class

Reply:
@spencergibb Cool, will do!